cells/lib/ops/mkStandardOCI.nix

```diff
@@ -68,19 +68,43 @@ in
       then debugOperable
       else operable;
 
-    setupLinks = cell.ops.mkSetup "links" [] ''
-      mkdir -p $out/bin
-      ${runtimeEntryLink}
-      ${debugEntryLink}
-      ${livenessLink}
-      ${readinessLink}
-    '';
+    setupLinks =
+      cell.ops.mkSetup "links" [
+        {
+          regex = "/bin";
+          mode = "0555";
+        }
+      ] ''
+        mkdir -p $out/bin
+        ${runtimeEntryLink}
+        ${debugEntryLink}
+        ${livenessLink}
+        ${readinessLink}
+      '';
+
+    setupBareMinimumEnv =
+      cell.ops.mkSetup "env" [
+        {
+          regex = "/tmp";
+          mode = "0777";
+        }
+      ] ''
+        mkdir -p $out/tmp
+        # Setup certificates
+        cp -r ${nixpkgs.cacert}/* $out
+      '';
+
+    setupSparseUser = cell.ops.mkUser {
+      user = "nobody";
+      group = "nogroup";
+      inherit uid gid;
+    };
```

Comment:
Similarly is there a penality within P95 for not exposing `withHome` & `withRoot`. Please expend the effort and[ look up the details](https://github.com/divnix/std/blob/main/cells/lib/ops/mkUser.nix#L30-L71) of what they do.